packages/fuels-contract/src/contract.rs

```diff
@@ -624,6 +661,43 @@ impl MultiContractCallHandler {
         self.get_response(receipts)
     }
 
+    /// Simulates a call without needing to specify the return type
+    async fn simulate_without_response(&self) -> Result<(), Error> {
+        let script = self.get_call_execution_script().await?;
+        let provider = self.wallet.get_provider()?;
+
+        script.simulate(provider).await?;
+
+        Ok(())
+    }
+
+    pub async fn set_estimated_tx_dependencies(
+        mut self,
+        max_attempts: Option<u64>,
```

Comment:
Are we going to reuse this function for estimating the number of Message Outputs as well? If so, will both outputs share a `max_attempts`?

Reply:
Yes, the idea is to re-use it. I thought about having x attempts per estimation target, but I guess that can be revisited in future PR's. Since this is not documented yet, we can afford to change the interface until we find the right combination.